examples/bcc/protocol_count/net_protocol.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/iovisor/gobpf/bcc"
+)
+
+const (
+	PROTOCOL_FUNC  = "protocol_count"
```

Comment:
How can I get this name list? I think 'protocol_count' is just like a filter's name, right？

Reply:
It is a function which is defined in net_protocol.c.